examples/async_clock/README.md

```diff
@@ -13,8 +13,8 @@ features:
 
 ## Running
 
-Run this application:
+Run a debug version of this application:
```

Comment:
```suggestion
Run this application with trunk development server:
```

I think it might be better to mention "trunk development server" than debug profile. For people coming from the JavaScript world, they are more familiar with the concept of "development server" than "debug profile". Trunk also provide some additional features when using their development server such as watching for changes and recompile.